lib/swoosh/email/smtp.ex

```diff
@@ -0,0 +1,90 @@
+defmodule Swoosh.Email.SMTP do
```

Comment:
I'm really not a fan of using the catch-all "Helpers" but I think I'd rather do that for now, so shall we move this module to `Swoosh.Adapters.SMTP.Helpers`, and I'm going to think about a better fitting name